hadoop-plugin/src/main/groovy/com/linkedin/gradle/spark/SparkSubmitHelper.groovy

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright 2015 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.linkedin.gradle.spark
+
+class SparkSubmitHelper {
+
+  static final EMPTY_STRING = ""
```

Comment:
You don't need a constant for the empty string. Normally, you would use a static final member variable to declare string constants you use more than once, but for the empty string, it's ok to simply put it throughout the code. The compiler will still optimize it down to one statically stored string.